data/eastern-hawk.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Eastern Hawk Solutions Pvt. Ltd.",
+    "establishedAt": 1533087376,
+    "location": "Urlabari-06, Morang, Nepal",
+    "website": "https://www.ehsnepal.com",
+    "email": "sales@ehsnepal.com",
+    "phone": " +977-9817340741",
```

Comment:
Does this place have a landline number?

Reply:
I reviewed it but unfortunately I couldn't find any landline numbers.